ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -234,9 +233,9 @@ module.exports = Marionette.LayoutView.extend({
     const queryContentView = this.queryView
       ? this.queryView
       : this.queryContent.currentView
-    if (!queryContentView.isValid()) {
-      announcement.announce(InvalidSearchFormMessage)
-      return
+    const validation = queryContentView.validate()
```

Comment:
Would it be good to keep both the original and new behavior in case there are other forms downstream that do not have a validate() function?